python-venv-util.sh

```diff
@@ -20,17 +20,18 @@ function in_python_venv() {
     python3 - ${PYTHON_VENV} <<EOF
 import sys
 import os
-expected_python_venv = sys.argv[1]
-current_python_venv = sys.prefix.split("/")[-1]
+from pathlib import PurePath
```

Comment:
Is my understanding correct that importing as `import pathlib` requires us to write `pathlib.PurePath` instead of just `PurePath`?

Reply:
yes